docs/source/polars-cloud/run/distributed-engine.md

```diff
@@ -11,8 +11,9 @@ significantly reduced, allowing users to process data at scale.
 <!-- dprint-ignore-start -->
 
 !!! info "Distributed engine is early stage"
+
     The distributed engine is still in the very early stages of development. Major performance improvements are planned for the near future. When an operation is not yet available in a distributed manner, Polars Cloud will execute it on a single node.
```

Comment:
I think we should mention that this might cause some queries to go out of memory. I think we are supposed to have the operations that are done in memory coloured red in our physical plan (?)